woocommerce/woocommerce
# add support for custom document and category titles

### Changes proposed in this Pull Request:

This PR adds support for custom post and category menu titles in the front matter in `.md` files in the `docs` folder.

The updated manifest includes quite a few changes from other PRs that have been merged. These changes will probably be easier to check/verify after the updated content has been ingested into a docs site.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Note the changes to `docs/user-experience/notices.md` are reflected in `docs-manifest.json`
2. Edit a `README.md` and another `.md` file
3. Add a `menu_title` to both files
4. Run the following:
```
pnpm --filter=monorepo-utils build 
pnpm utils md-docs create docs woocommerce
```
5. Check `manifest.json` in the root of the repo for the `menu_title` entries to be included



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
